Backends/CLX/frame-manager.lisp

```diff
@@ -58,64 +58,27 @@
 
 ;; Abstract pane lookup logic
 
-(defun find-first-defined-class (types)
-  (first
-   (remove-if #'null
-              (mapcar (lambda (class-name)
-                        (find-class class-name nil))
-                      types))))
-
-(defun find-symbol-from-spec (package-spec name-components)
-  (flet ((coerce-name-element (name-elt)
-           (typecase name-elt
-             (symbol (symbol-name name-elt))
-             (sequence (coerce name-elt 'string))
-             (t (princ-to-string name-elt)))))
-  (find-symbol
-   (apply #'concatenate 'string (mapcar #'coerce-name-element name-components))
-   package-spec)))
-
-(defun find-symbols (name-specs)
-  (remove-if #'null (mapcar #'(lambda (x) (find-symbol-from-spec (first x) (rest x))) name-specs)))
-
-(defun generate-standard-pane-specs (type)
-  (let ((mapping (get type 'climi::concrete-pane-class-name)))
-    `((,(symbol-package mapping) ,mapping)
-      (:climi ,mapping)
-      (:climi ,type #:-pane)
-      (:climi ,type))))
-
-(defun generate-clx-pane-specs (type)
-  (append
-   `((:clim-clx #:clx- ,type #:-pane)
-     (:clim-clx #:clx- ,type)
-     (:climi #:clx- ,type #:-pane)
-     (:climi #:clx- ,type))
-   (generate-standard-pane-specs type)))
```

Comment:
If I read the new version correctly, this naming scheme is no longer supported. Are there no pane classes which use this?

Reply:
classes which names are starting with #:clx- are produced with maybe-mirroring method. Since now we first resolve the clim class, then we optionally add #:clx-, and we *always* have this symbol in clim-clx package, this scheme is indeed unnecessary. It was already overboard with clim::clx- classes (scenario which never happened).